sdks/go/pkg/beam/testing/passert/equals_test.go

```diff
@@ -0,0 +1,84 @@
+package passert
```

Comment:
This file needs the apache license header.

Reply:
Added.